ndarray-rand/README.md

```diff
@@ -0,0 +1,68 @@
+ndarray-rand
+============
+
+Constructors for randomized arrays: `rand`'s integration with `ndarray`.
+
+Example
+=======
+
+Generate a 2-dimensional array with shape `(2,5)` and elements drawn from a uniform distribution
+over the `(0., 10.)` interval:
+
+```rust
+extern crate ndarray;
```

Comment:
```suggestion
```

Reply:
Forgot the README :man_facepalming: Thank you for spotting it :+1: